news/cross.rst

```diff
@@ -0,0 +1,24 @@
+**Added:**
+
+* Added partial support for cross compiling
```

Comment:
Can you detail what partial means?

Reply:
Unixes can cross compile for unixes. I didn't add support for other cross compiling to windows. Have to change a few templates to do that.

Reply:
K. We should add this note to the changelog.

Reply:
```suggestion
* Added partial support for cross compiling (Unixes can compile for other unixes only)
```